### ISSUE TITLE
Use native python http library

### ISSUE THREAD

**Issue by stphung** [autodesk-cloud/ochonetes#4]
Will enables windows to use the cli with low effort.

**Comment by opaugam** [comment]
do you care to elaborate my dear friend ? :)

**Comment by stphung** [comment]
Yeah, instead of calling out to the shell to `curl` we could use something like [requests](http://docs.python-requests.org/en/latest/user/quickstart/). Would keep the cli more self contained.

**Comment by opaugam** [comment]
yea i thought of it but we need something that is part of the standard libs. I'm not sure requests is (but we could probably go for urllib). Anyway who's still using windows as of 2015 ?!

**Comment by svoorhees** [comment]
:-) probably worth doing tho.  We will have windows users, whether we like it or not...
